collections/src/main/scala/strawman/collection/immutable/package.scala

```diff
@@ -0,0 +1,8 @@
+package strawman.collection
+
+package object immutable {
+  type StringOps = strawman.collection.StringOps
+  val StringOps = strawman.collection.StringOps
+  type StringView = strawman.collection.StringView
+  val StringView = strawman.collection.StringView
```

Comment:
Are these aliases used somewhere?

Reply:
Yes, because that's where these classes used to be. I think we should move them back (and had already done so) but the code was easier to rebase with them remaining in the collection package.